plugins/gitlab/src/class.ts

```diff
@@ -288,7 +297,11 @@ export class GitlabProjectApi extends GitlabApi {
     if (!projectGroup) {
       throw new Error('Parent DSO Project group has not been created yet')
     }
-    const projectsInGroup = await this.api.Groups.allProjects(projectGroup?.id, { perPage: 100 })
+    const projectsInGroup = await this.api.Groups.allProjects(projectGroup?.id, {
```

Comment:
le projectGroup peut ne pas avoir d'`id` si on throw une error si le `projectGroupe` is undefined ?